pyglet_tutorial/main.py

```diff
@@ -0,0 +1,71 @@
+import os
```

Comment:
File comment: Might recommend a more descriptive name since you might decide to do more than one pyglet tutorial